### PR TITLE
tidying new boxplot

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

File: instat/dlgBoxPlot.designer.vb [hunk 1]
```diff
@@ -27,20 +27,20 @@ Partial Class dlgBoxplot
         Me.lblByFactors = New System.Windows.Forms.Label()
         Me.lblBySecondFactor = New System.Windows.Forms.Label()
         Me.cmdBoxPlotOptions = New System.Windows.Forms.Button()
+        Me.chkHorizontalBoxplot = New System.Windows.Forms.CheckBox()
+        Me.ucrVariablesAsFactor = New instat.ucrVariablesAsFactor()
```
Comment (dannyparsons) [inline instat/dlgBoxPlot.designer.vb]: It's not a good idea if an instance of a class has the same name as the class itself. Can this be ucrVariablesToPlot or something?

**Comment by dannyparsons** [comment]
Can you delete the sub on line 90. It is not used anymore.

**Comment by dannyparsons** [comment]
Can you also change some of the labels: 
"By factor(s)" -> "By factor (optional)"
"By Second Factor" -> "By Second Factor (optional)"

**Comment by maxwellfundi** [comment]
@dannyparsons  this is done. Have alook

File: instat/dlgBoxPlot.designer.vb [hunk 2]
```diff
@@ -50,24 +51,24 @@ Partial Class dlgBoxplot
         Me.lblByFactors.AutoSize = True
         Me.lblByFactors.Location = New System.Drawing.Point(248, 143)
         Me.lblByFactors.Name = "lblByFactors"
-        Me.lblByFactors.Size = New System.Drawing.Size(60, 13)
+        Me.lblByFactors.Size = New System.Drawing.Size(97, 13)
         Me.lblByFactors.TabIndex = 2
         Me.lblByFactors.Tag = "By_factor_s_"
-        Me.lblByFactors.Text = "By factor(s)"
+        Me.lblByFactors.Text = "By factor (Optional)"
```
Comment (dannyparsons) [inline instat/dlgBoxPlot.designer.vb]: Capital F for factor.

**Comment by maxwellfundi** [comment]
@dannyparsons done!

**Comment by dannyparsons** [comment]
Great

**Pull request merged**